apps/pip/sbox/base/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I think you'll also need a cluster file here too due to this file : https://github.com/hmcts/sds-flux-config/blob/master/apps/pip/base/kustomize.yaml

looking for the cluster